dlp/snippets/deid_test.py

```diff
@@ -537,3 +537,52 @@ def test_deidentify_table_with_multiple_crypto_hash(capsys: pytest.CaptureFixtur
     assert "string_value: \"abbyabernathy1\"" not in out
     assert "my userid is abbyabernathy1" in out
     assert "aabernathy@example.com" not in out
+
+
+def test_deidentify_and_reidentify_table_with_fpe(capsys: pytest.CaptureFixture) -> None:
+    table_data = {
+        "header": ["employee_id", "date", "compensation"],
+        "rows": [
+            ["11111", "2015", "$10"],
+            ["11111", "2016", "$20"],
+            ["22222", "2016", "$15"],
+        ]
+    }
+
+    deid.deidentify_table_with_fpe(
+        GCLOUD_PROJECT,
+        table_data,
+        ["employee_id"],
+        alphabet='NUMERIC',
+        wrapped_key=WRAPPED_KEY,
+        key_name=KEY_NAME,
+    )
+
+    out, _ = capsys.readouterr()
+    assert "11111" not in out
+    assert "22222" not in out
+
+    response = out.split(":")[1:]
+
+    deid_col_id = response.index(' "employee_id"\n}\nheaders {\n  name')
```

Comment:
This is relying on python's default output formatting? If we need output to be in a specific format it would be better to do that in the sample code.

Reply:
Thank you for your response. I understand the concern, but to print table in known format, we will have to parse using the same code in sample itself. With that said, if the formatting of proto changes, this will affect the parser written in the sample code. So with either idea, we are dependent on the proto. Please let me know if I am missing anything here. Thank you!